C-Plus-Plus/cp/product_of_max_in_first_array_and_min_in_second.cpp

```diff
@@ -0,0 +1,80 @@
+/*Given two arrays:arr1[] and arr2[] of different size i.e.,no_of_ele_in_arr1 amd no_of_ele_in_arr2 respectively.
+Task :calculate the product of max element of first array i.e.,arr1[] and min element of second array i.e.,arr2[]
+Example:we are given two arrays arr1={1,2,3,4} and arr2={5,6,7,8,9} ,so the maximum element in array arr1 is 4 and minimum element in array arr2 is 5.Hence,the product of max element in array arr1 and min element in array arr2 is 4*5=20.
```

Comment:
Can you make it short, crisp and easy to read? Don't give examples here. Keep it at the bottom